upgrade.js

```diff
@@ -90,27 +90,23 @@ function main (knownUpgrades) {
 
   var elm = localFindBinary(
     'elm',
-    'ERROR: elm was not found on your PATH.  Make sure you have Elm 0.18 installed.\n' + howToInstallElm()
+    'ERROR: elm was not found on your PATH.  Make sure you have Elm 0.19 installed.\n' + howToInstallElm()
   )
 
-  var elmUsage = childProcess.execFileSync(elm)
-  var elmVersion = elmUsage.toString().split('\n')[0].split(' - ')[0].trim()
-  if (!elmVersion.match(/^Elm Platform 0\.18\./)) {
-    process.stderr.write('ERROR: Elm 0.18 required, but found ' + elmVersion + '\n' + howToInstallElm())
+  var elmUsage = childProcess.execFileSync(elm, [ '--version' ])
+  var elmVersion = elmUsage.toString().split('\n')[0]
+  if (!elmVersion.match(/^0\.19\./)) {
+    process.stderr.write('ERROR: Elm 0.19 required, but found ' + elmVersion + '\n' + howToInstallElm())
```

Comment:
Could use template strings instead here

Reply:
I don't understand the difference between all the different versions of nodejs...  are template strings supported in all versions that people are likely to have?  Apparently the default nodejs version on Travis-CI is node 0.10, which didn't seem to support template strings.

Reply:
This has some stats on what versions of node people are using: https://nodesource.com/node-by-numbers

Generally, people who use Elm also have modern versions of Node installed. I'm not sure if there's a good way to identify that. At least elm-test requires Node > 4.7: https://github.com/rtfeldman/node-test-runner/blob/master/package.json#L7